.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What are we updating (eg. update button destructive color)?'
+        default: ''
+        required: true
+      jira:
+        description: 'What is the related JIRA (eg. SW-1234)?'
+        default: ''
+        required: true
+      email:
+        description: 'Your terraformation email'
+        default: ''
+        required: true
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Create branch
+        run: git checkout -b `echo design-workflow-${{ github.actor }}-${{ github.event.inputs.jira }}-\`date +"%y-%m-%d"\`  | sed "s/ /-/g"`
```

Comment:
A couple shell things: for command substitution, `$()` nests more cleanly than backticks (no need to escape the nested commands). And `tr` does global character replacement slightly more succinctly than `sed`:
```
run: git checkout -b $(echo design-workflow-${{ github.actor }}-${{ github.event.inputs.jira }}-$(date +%y-%m-%d) | tr ' ' -)
```